Documentation/ceph-object.md

```diff
@@ -285,15 +285,15 @@ metadata:
 spec:
   ports:
   - name: rgw
-    port: 80
+    port: 8080
     protocol: TCP
-    targetPort: 80
+    targetPort: 8080
```

Comment:
What if you update to port 8443 as in the loadbalancer example previously referenced?